config/manager/manager_image_patch.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        # Change the value of image field below to your controller image URL
+        - image: ghcr.io/nutanix-cloud-native/cluster-api-provider-nutanix/controller:latest
```

Comment:
can we make sure to remove the latest tagged image in github registry? else sometimes local dev pull that if we have not done push kind https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pkgs/container/cluster-api-provider-nutanix%2Fcontroller/216366670?tag=latest

Reply:
no, we will not be removing the latest tagged image; that breaks away from convention.

Reply:
i have getting error on fresh make deploy on a kind cluster where capx controller crashes with invalid flag --metrics-... and expects --diagnostics..... i feel that its because of stale latest image in cloud. do you know if its getting updated at all?

Reply:
latest image in production is likely because 'latest' refers to 1.3.4. In 1.4.x (and main) we have different options i.e. diagnostics-addr. I'm wondering if it has something to do with that. In that case imo, it's make deploy target that needs fixing and not the image.